### PR TITLE
Add compute schemas generated from compute swagger documents

### PR DESCRIPTION
**Pull request by daschult**

### CODE REVIEW

**Comment by wenwu449** [comment]
Similar to Microsoft.Storage schema, can we only update for the new apiVersion 2016-03-30?

**Comment by daschult** [comment]
Removed.

File: schemas/2016-03-30/Microsoft.Compute.json [hunk 1]
```diff
@@ -0,0 +1,1976 @@
...
+            }
+          ],
+          "description": "Gets or sets a list containing reference to all Virtual Machines created under this Availability Set."
+        },
+        "statuses": {
+          "oneOf": [
+            {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/InstanceViewStatus"
```
Comment (yangl900) [inline schemas/2016-03-30/Microsoft.Compute.json]: I thought this is readonly property?

Comment (daschult) [inline schemas/2016-03-30/Microsoft.Compute.json]: It wasn't marked as "readOnly" in the Microsoft.Compute swagger document, so I have no way of knowing. I can remove it for now.

**Comment by daschult** [comment]
Can we merge this pull request?

File: schemas/2016-03-30/Microsoft.Compute.json [hunk 2]
```diff
@@ -0,0 +1,1962 @@
...
+            {
+              "$ref": "#/definitions/VirtualMachineProperties"
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "resources": {
```
Comment (wenwu449) [inline schemas/2016-03-30/Microsoft.Compute.json]: is this in the correct level? it should be under the "properties", right?

**Comment by philon-msft** [comment]
@wenwu449, can this be merged now?

**Pull request closed without merging**